File: lectures/STSDemo/src/com/shawn/stsdemo/Dog.java
package com.shawn.stsdemo;

public class Dog {
	private String name;
	private int age;
	private String hairColor;

	public Dog() {}

	public Dog(String name, int age, String hairColor) {
		this.name = name;
		this.age = age;
		this.hairColor = hairColor;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public String getHairColor() {
		return hairColor;
	}

	public void setHairColor(String hairColor) {
		this.hairColor = hairColor;
	}

}

File: lectures/W1D2_JavaOOP/Dog.py
class Dog:
    def __init__(self, name, age, hair_color):
        self.name = name
        self.age = age
        self.hair_color = hair_color

    @classmethod
    def getAllDogs():
        pass

    def doSomething(): 
        pass


d1 = Dog()

d1.doSomething()

Dog.getAllDogs()